CHANGELOG.md

```diff
@@ -0,0 +1,158 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+### Added
+- Added `needs_review` filter for `merge_labels`.
+- Added release_criteria
+- Added modularized config structure
+- New `join_observations` utility
+- Ability to configure the number of extractors in Makefile
+- Feature Lists for sqwiki, kowiki, fiwiki, rowiki, zhwiki and glwiki
+- Code of Conduct
+
+### Fixed
+- `extractors` typo in generated parameters
+- Flaky config test
+- Notebook fixes for revscoring 2.x
+- Silence mediawiki API errors in `autolabel`.
+- `merge_labels` degenerate bug.
+- Argument order for `merge_label` utility
+- Minor issue with autolabel mw session args
+
+### Changed
+- Bumped revscoring version to 2.5.1
+- Refactored templates to be more modular
+- Makefile now reads `trusted_edits` config.
+- Use `min_samples_leaf` as a tuning param for GB.
+- Interpret boolean string literals in label merge.
+- Make option of auto-labeled dataset optional in `merge_labels`.
+- Standardized `generate_make` utility and args.
+- Use informals and badwords for wikidata comment parsing.
+- Assume goodfaith in `fetch_labels`.
+- Exclusions for huwiki badwords/informals
+- Added `filter_rate_at_recall` metrics back to Makefile
+- Migrate from nosetest to pytest
+
+### Removed
+- XGBoost requirement
+
+## [v0.4.2] - 2017-01-07
+
+### Changed
+- `test_statistics` model param for damaging and goodfaith filtering
```

Comment:
A few things that look like they are missing: 

Added
- Adds feature_lists for svwiki https://github.com/wikimedia/editquality/commit/465faa52bd14863ced577cffd7a01057b3a12120
- Adds feature_lists for cswiki https://github.com/wikimedia/editquality/commit/8de31d8a30e0471caaf11032a65e1eafc61fc1d2
- Adds `--filter` param to `fetch_labels` utility https://github.com/wikimedia/editquality/commit/a5596630f894483f90adcbf2859bc319618dcead
- Adds "about.py" variables (__version__, __author__, etc.) https://github.com/wikimedia/editquality/commit/6c2ed59329f45e7e38a92be9d7cebe5d93401cef

Changed
- Utilities now use JSON-lines format https://github.com/wikimedia/editquality/commit/7926df7e37d9fbaf0a3f58638189ad6c2ed5b4cf#diff-17be14e81cadfdb3008ecb3797e57bb2
- Renamed "label_reverted" to "autolabel" https://github.com/wikimedia/editquality/commit/7926df7e37d9fbaf0a3f58638189ad6c2ed5b4cf#diff-17be14e81cadfdb3008ecb3797e57bb2
- Bumped revscoring requirement to 1.2.8 https://github.com/wikimedia/editquality/commit/f9739b7c5589e9a748a87f749384b82d6c252fae

Fixed
- Data normalization issues in autolabel utility https://github.com/wikimedia/editquality/commit/eb1028c2e1a93d908249e61177ac6ec0990bb992

Removed
- XGBoost from models to tune https://github.com/wikimedia/editquality/commit/c20093b811009d3e313dbbfb2727dbf7323a3fc6

Reply:
added!